Ginger/GingerCore/Drivers/Appium/SeleniumAppiumDriver.cs

```diff
@@ -475,64 +529,72 @@ private DriverOptions GetCapabilities()
-                    if (!string.IsNullOrEmpty(DeviceID))
-                        driverOptions.AddAdditionalCapability("udid", DeviceID);
-
-                    //Tested application capabilities 
                     driverOptions.AddAdditionalCapability("browserName", "chrome");
                     //capabilities.SetCapability("chromedriverExecutable", ""); //The absolute local path to webdriver executable (if Chromium embedder provides its own webdriver, it should be used instead of original chromedriver bundled with Appium); exp: /abs/path/to/webdriver
                     //capabilities.SetCapability("chromeOptions", ""); //Allows passing chromeOptions capability for chrome driver. For more information see chromeOptions: https://sites.google.com/a/chromium.org/chromedriver/capabilities
                     break;
 
                 case eSeleniumPlatformType.iOSBrowser:
-                    //Appium capabilities
-                    driverOptions.AddAdditionalCapability("newCommandTimeout", SERVER_TIMEOUT_SEC); //How long (in seconds) Appium will wait for a new command from the client before assuming the client quit and ending the session
-
-                    //Tested device capabilities
-                    driverOptions.AddAdditionalCapability("deviceName", DeviceName);
-                    driverOptions.AddAdditionalCapability("platformVersion", DevicePlatformVersion);//Mobile OS version 
-                    if (!string.IsNullOrEmpty(DeviceID))
-                        driverOptions.AddAdditionalCapability("udid", DeviceID);
-
                     //Tested application capabilities 
                     driverOptions.AddAdditionalCapability("browserName", "safari");
                     //capabilities.setCapability("safariInitialUrl", "");  //(Sim-only) (>= 8.1) Initial safari url, default is a local welcome page
                     driverOptions.AddAdditionalCapability("safariAllowPopups", false);  //(Sim-only) Prevent Safari from showing a fraudulent website warning. Default keeps current sim setting.
                     //capabilities.setCapability("safariOpenLinksInBackground", false);  //(Sim-only) Whether Safari should allow links to open in new windows. Default keeps current sim setting.
-                    if (!string.IsNullOrEmpty(InstalledAppBundleID))
-                        driverOptions.AddAdditionalCapability("bundleId", InstalledAppBundleID);
                     break;
             }
 
+            //Generic capabilities
+            //driverOptions.AddAdditionalCapability("newCommandTimeout", SERVER_TIMEOUT_SEC); //How long (in seconds) Appium will wait for a new command from the client before assuming the client quit and ending the session
+            driverOptions.AddAdditionalCapability("newCommandTimeout", NewCommandTimeout.ToString());            
+            if (string.IsNullOrEmpty(AutomationName) == false)
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6113827)